packages/legend-application-studio/src/stores/EditorGraphState.ts

```diff
@@ -186,6 +186,22 @@ export class EditorGraphState {
     return [this.error, ...this.warnings].filter(isNonNullable);
   }
 
+  removeDependencyProblems(
+    problems: Problem[] | CompilationWarning[],
+  ): Problem[] | CompilationWarning[] {
+    return problems.filter((problem) => {
+      if (problem.sourceInformation) {
```

Comment:
I don't see how this filtering work in form mode where the problem probably doesn't have any source information at all

Reply:
😔